packages/async-popup/.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
+.merlin
+.bsb.lock
+npm-debug.log
```

Comment:
let's add the .bs.js to ignore too?

Reply:
Alright. I'll add it to the template too

Reply:
thx